build.cake

```diff
@@ -107,6 +107,17 @@ Task("NuGetPush")
     });
 });
 
+Task("PowerShellPack")
+    .IsDependentOn("Test")
+    .Does(() =>
+{
+    var srcDir = projectFilePowerShell.Path.GetDirectory().FullPath;
```

Comment:
Do we really need to copy the entire directory into artifacts?
It seems we need only the Grok.psd1 and Grok.Net.PowerShell.dll.
What do you think about this:

```suggestion
    var psModuleFiles = GetFiles("./src/Grok.Net.PowerShell/Grok.psd1")
        .Concat(GetFiles("./src/Grok.Net.PowerShell/**/Grok.Net.PowerShell.dll"));
    CopyFiles(psModuleFiles, artifactsDir);
```

Reply:
Yes we need the whole directory, and it has to be copied to a separated folder as required by `Publish-Module` cmdlet.